app/helpers/meta_tags_helper.rb

```diff
@@ -0,0 +1,10 @@
+module MetaTagsHelper
+  def default_meta_tags
```

Comment:
there's no need to create more and more helper modules.. it's bad for performance,
meta tags are parts of the app components, so `ApplicationHelper` is not really a bad place for this helper